docker-compose.yaml

```diff
@@ -6,6 +6,8 @@ services:
   # RD-WATCH
   rdwatch:
     image: ghcr.io/resonantgeodata/rd-watch/rdwatch:latest
+    build: 
```

Comment:
What is the purpose of this change? It looks like it will change the semantics of the `image` property here (https://stackoverflow.com/a/34316224)

Reply:
thanks, that was left over from when I was doing local builds.  I'll remove it.